CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [1.22](https://github.com/brave/brave-ios/releases/tag/v1.22)
+
+ - Added Brave Today news feed on the New Tab Page. ([#2863](https://github.com/brave/brave-ios/issues/2863))
+ - Removed redundant rewards setting to address user confusion. ([#3009](https://github.com/brave/brave-ios/issues/3009))
+ - Updated default search engine to Yandex for new installations in certain regions. ([#3022](https://github.com/brave/brave-browser/issues/#3022))
+ - Fixed crash when opening links from other applications while Brave set as default browser. ([#3041](https://github.com/brave/brave-browser/issues/#3041))
+ - Fixed crash when adding or editing bookmarks in certain cases. ([#3032](https://github.com/brave/brave-browser/issues/#3032))
```

Comment:
Update URL, change "brave-browser" to "brave-ios" and remove the # in the URL

Reply:
++ on this; should be https://github.com/brave/brave-ios/issues/3032